task/sql-tasks.js

```diff
@@ -199,7 +328,26 @@ async function task_1_14(db) {
  *
  */
 async function task_1_15(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+    SELECT

```

Comment:
Try to do it like this

select
    count(monthcondition) as month
FROM Orders
Where Year = 1997